cmd_set.go

```diff
@@ -660,29 +667,40 @@ func (m *Miniredis) cmdSscan(c *server.Peer, cmd string, args []string) {
 	withTx(m, c, func(c *server.Peer, ctx *connCtx) {
 		db := m.db(ctx.selectedDB)
 		// return _all_ (matched) keys every time
-
-		if opts.cursor != 0 {
-			// invalid cursor
-			c.WriteLen(2)
-			c.WriteBulk("0") // no next cursor
-			c.WriteLen(0)    // no elements
-			return
-		}
 		if db.exists(opts.key) && db.t(opts.key) != "set" {
 			c.WriteError(ErrWrongType.Error())
 			return
 		}
-
 		members := db.setMembers(opts.key)
 		if opts.withMatch {
 			members, _ = matchKeys(members, opts.match)
 		}
-
+		low := opts.cursor
+		end := low + opts.count
+		// validate end is correct
+		if opts.count == 0 || end > len(members) {
```

Comment:
The behavior if opts.count from a redis I tried running locally was to error:

```
127.0.0.1:6379> SSCAN foo "0" COUNT 0
(error) ERR syntax error
127.0.0.1:6379> SSCAN foo "0" COUNT 1
1) "0"
2) 1) "1"
   2) "2"
   3) "3"
```

Reply:
Done, also added a test to confirm it.